app/javascript/modal.js

```diff
@@ -1,44 +1,44 @@
-// Make the DIV element draggable:
-dragElement(document.getElementById("modal-container"));
+// // Make the DIV element draggable:
+// dragElement(document.getElementById("modal-container"));
 
-function dragElement(elmnt) {
-  let pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
+// function dragElement(elmnt) {
+//   let pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
   
-  const header = document.getElementById(elmnt.id + "-header");
-  if (header) {
-    // Move the DIV from the header
-    header.onmousedown = dragMouseDown;
-  } else {
-    // Move the DIV from anywhere in the DIV
-    elmnt.onmousedown = dragMouseDown;
-  }
+//   const header = document.getElementById(elmnt.id + "-header");
+//   if (header) {
+//     // Move the DIV from the header
+//     header.onmousedown = dragMouseDown;
+//   } else {
+//     // Move the DIV from anywhere in the DIV
+//     elmnt.onmousedown = dragMouseDown;
+//   }
 
-  function dragMouseDown(e) {
-    e = e || window.event;
-    e.preventDefault();
-    // Get the cursor position at startup
-    pos3 = e.clientX;
-    pos4 = e.clientY;
-    document.onmouseup = closeDragElement;
-    document.onmousemove = elementDrag;
-  }
+//   function dragMouseDown(e) {
+//     e = e || window.event;
+//     e.preventDefault();
+//     // Get the cursor position at startup
+//     pos3 = e.clientX;
+//     pos4 = e.clientY;
+//     document.onmouseup = closeDragElement;
+//     document.onmousemove = elementDrag;
+//   }
 
-  function elementDrag(e) {
-    e = e || window.event;
-    e.preventDefault();
-    // Calculate the new cursor position
-    pos1 = pos3 - e.clientX;
-    pos2 = pos4 - e.clientY;
-    pos3 = e.clientX;
-    pos4 = e.clientY;
-    // Set the element's new position
-    elmnt.style.top = (elmnt.offsetTop - pos2) + "px";
-    elmnt.style.left = (elmnt.offsetLeft - pos1) + "px";
-  }
+//   function elementDrag(e) {
+//     e = e || window.event;
+//     e.preventDefault();
+//     // Calculate the new cursor position
+//     pos1 = pos3 - e.clientX;
+//     pos2 = pos4 - e.clientY;
+//     pos3 = e.clientX;
+//     pos4 = e.clientY;
```

Comment:
A bit confused on changes happening in this file. Seems like you added a bunch of comments?

Reply:
this was for the pop up stuffs, i didnt know i want to delete it or not

Reply:
sounds good